.github/workflows/Deploy_web_site_with_ssh.yml

```
@@ -0,0 +1,34 @@
+name: Deploy_web_site_with_ssh
+
+# read-only repo token
+# no access to secrets
+on:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Change BASE_PATH
+        run: yq -i '.JB.BASE_PATH = "https://cgal.geometryfactory.com/~${{ secrets.SSH_USER }}/cgal-web"' _config.yml
```

It would better be a variable. See [the documentation of actions variables](https://docs.github.com/en/actions/learn-github-actions/variables#defining-configuration-variables-for-multiple-workflows).

```suggestion
        run: yq -i '.JB.BASE_PATH = "${{ vars.CGAL_WEB_SITE_URL }}"' _config.yml
```